packaging/installer/methods/kubernetes.md

```diff
@@ -19,94 +20,119 @@ The Helm chart installs one `parent` pod for storing metrics and managing alarm
+- The [Helm package manager](https://helm.sh/) v3.0.0 or newer on the same administrative system.
+- A Netdata Cloud account with a Space to claim the cluster to.
 
-## Install the Netdata Helm chart
+## Deploy Netdata on your Kubernetes Cluster
 
-We recommend you install the Helm chart using our Helm repository. In the `helm install` command, replace `netdata` with
-the release name of your choice.
+To start [Kubernetes monitoring](https://github.com/netdata/netdata/blob/master/docs/cloud/visualize/kubernetes.md), you must first install Netdata, and then
+[connect](https://github.com/netdata/netdata/blob/master/claim/README.md) your Kubernetes cluster to [Netdata Cloud](https://app.netdata.cloud). The connection process securely
+connects your Kubernetes cluster to stream metrics data to Netdata Cloud, enabling Kubernetes-specific visualizations
+like the health map and time-series composite charts.
 
-```bash
-helm repo add netdata https://netdata.github.io/helmchart/
-helm install netdata netdata/netdata
-```
+<Tabs groupId="installation_type">
+<TabItem value="new_installations" label="New Installations">
 
-Run `kubectl get services` and `kubectl get pods` to confirm that your cluster now runs a `netdata` service, one
-parent pod, and multiple child pods.
+#### Steps
 
-You've now installed Netdata on your Kubernetes cluster. Next, it's time to opt-in and enable the powerful Kubernetes
-dashboards available in Netdata Cloud.
+1. Add the Netdata Helm chart repository by running:
 
-## Connect your Kubernetes cluster to Netdata Cloud
+  ```bash
+  helm repo add netdata https://netdata.github.io/helmchart/
+  ```
 
-To start [Kubernetes monitoring](https://learn.netdata.cloud/docs/cloud/visualize/kubernetes/), you must first
-[connect](/claim/README.md) your Kubernetes cluster to [Netdata Cloud](https://app.netdata.cloud). The connection process securely
-connects your Kubernetes cluster to stream metrics data to Netdata Cloud, enabling Kubernetes-specific visualizations
-like the health map and time-series composite charts.
+2. To install Netdata using the `helm install` command, run:
 
-### New installations
+  ```bash
+  helm install netdata netdata/netdata 
+  --set parent.claiming.enabled="true" 
+  --set parent.claiming.token=YOUR_CLAIM_TOKEN 
+  --set parent.claiming.rooms=YOUR_ROOM_ID_A,YOUR_ROOM_ID_B 
+  --set child.claiming.enabled=true 
+  --set child.claiming.token=YOUR_CLAIM_TOKEN 
+  --set child.claiming.rooms=YOUR_ROOM_ID_A,YOUR_ROOM_ID_B
+  ```
 
-First, find the script to run an `helm install` command. You can get it by clicking on your Space's dropdown, then **Manage your Space**.
-Click the **Nodes** tab and select the environment your node is running, in this case **kubernetes**, to reveal the script for your Space in Netdata Cloud. You need the `TOKEN`
-and `ROOM` values.
+  :::note
+  If you plan to also Claim the node to Netdata Cloud,
```

Comment:
Well, it's a bit strange to give them a command that has the tokens and then say that IF they want to connect to cloud they should xyz. 

This isn't the same as the one line installer though, where we give them additional options in this doc. The full reference is actually on github.com/netdata/helmchart/README.md 
So:
- Give them the simple the helm install command. 
- For claiming, tell them to get their info directly from "Add nodes" in app.netdata.cloud. 
- For more installation options direct them to helmchart/README  (we need to add helmchart to learn as well, obviously).